reference_docs/YAML_models/iec_document_attributes.yaml

```diff
@@ -0,0 +1,111 @@
+version: v3.4.5 #Verify versions
+
+inherits_from:
+  # Also may be referred to as “parent flavor”
+  flavor: standoc
+  version: v1.x.x
+
+document_attributes:
+#--- Parts and subparts  ---#
+  partnumber:
+    tags: [""]
+    title: "Part number"
+    description: |
+      "IEC documents can have both parts and subparts, unlike ISO documents which only supports parts.\n
+      Both `{part}` and `{subpart}` only accept numerals.".
+    value_spec: "other: `{part}` or `{part}`-`{subpart}`"
+
+#--- Document relations ---#
+  translated-from:
```

Comment:
Ah. What I'd like to see is only a subset of attributes used for override: i.e. the definition is overridden, but all the other attributes are not. If we do that, then we would need to override the  script behaviour of populating all fields by default, for those attributes that are already defined elsewhere, so that an override is only partially populated. That's cleaner; it's also harder to manage. The alternative is to repeat the definition entirely for an inherited attribute, which is easier to manage, but bad info management practice.

I prefer the former, but this is up to whoever is going to realise the inheritance here. Who is that?

Reply:
I agree with your concerns, and would also prefer only overriding the description. We can discuss who should take care of this in our upcoming bi-weekly meeting?